app/mailers/proposal_mailer.rb

```diff
@@ -1,4 +1,5 @@
 class ProposalMailer < ApplicationMailer
+  DMMT_USER_GUIDE_URL = "https://wiki.earthdata.nasa.gov/display/ED/Draft+MMT+%28dMMT%29+User%27s+Guide"
   def proposal_submitted_notification(user, short_name, version, id, request_type)
```

Comment:
nitpick - think there should be a space after the constant (don't change unless you have to make other changes)